### PR TITLE
solution

### PR DESCRIPTION
**Pull request by deelray**
[DEMO LINK](https://deelray.github.io/react_sum/)

### CODE REVIEW

**Review by olesiatk — APPROVED** [review]
Well done!

File: src/components/Sum.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a = 2, b = 1 }) => (
```
Comment (aionova) [inline src/components/Sum.js:4]: If you indicate that all of your props are required, default values are redundant

**Review by aionova — CHANGES_REQUESTED** [review]
Good start!

**Review by aionova — APPROVED** [review]
Awesome!

**Pull request closed without merging**